software/azimutt.yml

```
@@ -0,0 +1,17 @@
+name: "Azimutt"
+website_url: "https://azimutt.app"
+source_code_url: "https://github.com/azimuttapp/azimutt"
+description: "Visual database exploration made for real world databases (big and messy). Explore your database schema as well as data, document them, extend them and even get analysis and guidelines."
+licenses:
+  - MIT
+platforms:
+  - Javascript
+  - Elixir
+  - Nodejs
+  - Docker
+tags:
+  - Database Management
+  - Software Development
+  - Software Development - IDE & Tools
```

Software development is a category heading and is not needed.

```suggestion
tags:
  - Database Management
  - Software Development - IDE & Tools
```